src/client/components/meal/AddMeal.js

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from "react";
+
+function AddMeal({ addMeal }) {
+  const initialstateForMeal = {
```

Comment:
This could be declared outside of the component, so it's not defined on each render.